src/automation/atm_dicts.py

```diff
@@ -86,9 +111,9 @@
     s3_prefix='observations/reanalysis/amv/merged/%Y/%m/bufr/',
     bufr_files='gdas.%z.satwnd.tm00.bufr_d',
     nceplibs_cmd=au.NCEPLIBS_SINV,
-    platform=au.CLEAN_PLATFORM,
+    platform=au.PRIVATE_EUMETSAT_PLATFORM,
```

Comment:
Just checking, this is the only variable that has the platform updated to EUMETSAT, but there's nothing in the variable name or file path to indicate that it is EUMETSAT. Is this data stored in the private bucket?

Reply:
Yes, this is a weird one, the "merged" part is what comes from eumetsat/esa so this is one of the data streams that's going to be private. Good question though